docs/metadata/prophecy-build-tool.md

```diff
@@ -67,14 +69,23 @@ export DATABRICKS_TOKEN="exampledatabrickstoken"
 
 #### Building Pipelines and deploying Jobs
 
+PBT can build and deploy Jobs inside your Prophecy project to the Databricks environment defined by the `DATABRICKS_HOST` and `DATABRICKS_TOKEN`
+environment variables.
+
+Since v1.0.3 PBT supports new input parameters that are used to determine the DBFS path where your project's artifacts would
+be uploaded. These are the `--release-version` and `--project-id` parameters which would be used to replace the `__PROJECT_RELEASE_VERSION_ PLACEHOLDER__` and `__PROJECT_ID_PLACEHOLDER__` placeholders that would already be present in your Job's definition file
```

Comment:
We should add an example of using the new tag from Prophecy release

Reply:
This does not create any new tag.  __PROJECT_RELEASE_VERSION_ here is to just construct the final path where the jar goes.